include/lsst/afw/table/Exposure.h

```diff
@@ -381,7 +381,7 @@ class ExposureCatalogT : public SortedCatalogT<RecordT> {
      *
      *  The returned array's records are shallow copies, and hence will not in general be contiguous.
      */
-    ExposureCatalogT<RecordT> subset(ndarray::Array<bool const,1> const & mask) const {
+    ExposureCatalogT subset(ndarray::Array<bool const,1> const & mask) const {
```

Comment:
Good question. I did it to improve uniformity with most of the rest of our code and internal self-consistency. I found it confusing to have the same return type spelled two different ways within the same file. I chose the shorter version because @TallJimbo prefers it and because more of our code uses it.